src/wagtailmedia/blocks.py

```diff
@@ -6,8 +6,14 @@
 from django.template.loader import render_to_string
 from django.utils.functional import cached_property
 
+from wagtail import VERSION as WAGTAIL_VERSION
 from wagtail.admin.compare import BlockComparison
-from wagtail.core.blocks import ChooserBlock
+
+
+if WAGTAIL_VERSION >= (3, 0):
```

Comment:
given we have `install_requires=["wagtail>=4.1", "Django>=3.2"],` in setup.py... these conditionals are obsolete...